packages/manager/modules/pci/src/projects/project/storages/cold-archive/cold-archive.routing.js

```diff
@@ -24,7 +25,26 @@ export default /* @ngInject */ ($stateProvider) => {
               },
         ),
     resolve: {
-      guides: () => GUIDES,
+      guides: /* @ngInject */ (coreConfig, $translate) =>
+        reduce(
+          GUIDES,
+          (list, guide) => [
+            ...list,
+            {
+              ...guide,
+              title: $translate.instant(
+                `pci_projects_project_storages_cold_archives_guides_${guide.id}_title`,
+              ),
+              description: $translate.instant(
+                `pci_projects_project_storages_cold_archives_guides_${guide.id}_description`,
+              ),
+              link:
+                guide.links[coreConfig.getUserLanguage()] ||
```

Comment:
Are we sure about this ?
Don't we usually use the `ovhSubsidiary` ?
```suggestion
                guide.links[coreConfig.getUser().ovhSubsidiary] ||
```